### ISSUE TITLE
Draw + Globe projection issue at small zoom levels [Bug]

### ISSUE THREAD

**Issue by arthurdjn** [visgl/react-map-gl#2030]
### Description

I have an issue while using `mapbox-gl-draw` and `react-map-gl` draw together. They both work fine with the default options (just like in the [example](https://github.com/visgl/react-map-gl/tree/master/examples/draw-polygon)) but when I try to change the `projection="globe"` prop in the `Map` component, then I can't 'finish' my drawing - see video to better understand my issue. When I zoom out at ~3 zoom level, everything works as expected.

![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/48992917/198888567-2fb45820-a5b4-43c7-94b6-70c56f43ef49.gif)

What I have tested:
* I tried using a minimal `DrawControl` just like in the [docs example](https://visgl.github.io/react-map-gl/docs/api-reference/use-control) but did not work.
* Also  everything works as expected when I use the [example from mapbox](https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-draw/) with `projection: "globe"` option.

### Expected Behavior

It should also be possible to 'finish' the polygon by clicking on the first point. The last two 'points' should also appear when drawing a polygon, just like at higher zoom level or with normal projection.

### Steps to Reproduce

Just change the following in [`app.tsx`](https://github.com/visgl/react-map-gl/blob/master/examples/draw-polygon/src/app.tsx):

```diff
...
  <Map
    initialViewState={{
      longitude: -91.874,
      latitude: 42.76,
+     zoom: 15
    }}
    mapStyle="mapbox://styles/mapbox/satellite-v9"
    mapboxAccessToken={TOKEN}
+   projection="globe"
  >
...
```

### Environment

- React: react@18.2.0
- Framework version: react-map-gl@7.0.19
- Map library: mapbox-gl@2.0.1
- Browser: Chrome Version 106.0.5249.119
- Browser: Firefox Version 106.0.2
- OS: Windows 11

**Comment by kylerummens** [comment]
I'm seeing the same issue. Were you able to find a solution?

**Comment by arthurdjn** [comment]
> I'm seeing the same issue. Were you able to find a solution?

Looks like this issue is fixed in version 1.7.3 ([#2229](https://github.com/visgl/react-map-gl/pull/2229)).
Try to use latest version from react map gl, mapbox and mapbox-gl-draw.

Closing this issue.
